hub/api/dataset.py

```diff
@@ -167,7 +171,28 @@ def __init__(
             self._schema = hub.schema.deserialize.deserialize(self.meta["schema"])
             self._meta_information = self.meta.get("meta_info") or dict()
             self._flat_tensors = tuple(flatten(self._schema))
+            try:
+                version_info = pickle.loads(fs_map["version.pkl"])
+                self._branch_node_map = version_info["branch_node_map"]
```

Comment:
switch to get

Reply:
@imshashank I'm leaving direct access instead of get because we want to catch all possible exceptions here, which get will hide. Any exception while loading should just lead to the dataset not having version control and behaving as a 1.2.x dataset

Reply:
@AbhinavTuli we no longer want to catch an arbitrary Exception, we want to delegate everything to exceptions.py since we are adding telemetry. So please import the exception from the exceptions and name it accordingly.

Reply:
Fixed